tests/testthat/test-derive_vars_dtm_to_dt.R

```diff
@@ -40,3 +41,43 @@ test_that("Convert a complete -- DTM into a date object", {
     keys = c("STUDYID", "USUBJID", "TRTSDTM", "ASTDTM", "AENDTM")
   )
 })
```

Comment:
same here 
- line 19: expected_output; I would remove TRTSDT, ASTDT, AENDT from the tribble and just create them in the mutate step from the DTM variable. I would also probably make 1 subject per row otherwise it looks strange to have multiple trt start for the same pt 
- line41: if you make 1 subject per row then the keys would need to be updated